### PR TITLE
Fix example_dags and system tests "source" links in documentation

### PR DESCRIPTION
**Pull request by potiuk**
This is the result of running the scripts from
https://github.com/apache/airflow/pull/24389

Fixes: https://github.com/apache/airflow/issues/23632

### CODE REVIEW

File: docs-archive/apache-airflow-providers-airbyte/2.1.1/operators/airbyte.html [hunk 1]
```diff
@@ -609,7 +609,7 @@ <h2>Using the Operator<a class="headerlink" href="#using-the-operator" title="Pe
 return the <code class="docutils literal notranslate"><span class="pre">job_id</span></code> that should be pass to the AirbyteSensor.</p>
 <p>An example using the synchronous way:</p>
 <div class="example-block-wrapper docutils container">
-<p class="example-header example-header--with-button"><span class="example-title">airflow/providers/airbyte/example_dags/example_airbyte_trigger_job.py</span><a class="example-header-button viewcode-button reference internal" href="../_modules/airflow/providers/airbyte/example_dags/example_airbyte_trigger_job.html"><span class="viewcode-link">[source]</span></a></p>
+<p class="example-header example-header--with-button"><span class="example-title">airflow/providers/airbyte/example_dags/example_airbyte_trigger_job.py</span><a class="example-header-button viewcode-button reference internal" href="https://github.com/apache/airflow/tree/providers-airbyte/2.1.1/airflow/providers/airbyte/example_dags/example_airbyte_trigger_job.py" target="_blank"><span class="viewcode-link">[source]</span></a></p>
```
Comment (ashb) [inline docs-archive/apache-airflow-providers-airbyte/2.1.1/operators/airbyte.html:612]: But this file exists https://airflow.apache.org/docs/apache-airflow-providers-airbyte/2.1.1/_modules/airflow/providers/airbyte/example_dags/example_airbyte_trigger_job.html

Why aren't we linking to it?

**Review by ashb — CHANGES_REQUESTED** [review]

**Comment by potiuk** [comment]
Hey @ashb  - re-rerun the scripts on base changes and airbyte is gone. I think it could be a remnant of the initial tests I've done. I'v optimized the scripts as well (less throttling possibility).

File: docs-archive/apache-airflow-providers-apache-beam/4.0.0/operators.html [hunk 2]
```diff
@@ -616,7 +616,7 @@ <h1>Apache Beam Operators<a class="headerlink" href="#apache-beam-operators" tit
 <div class="section" id="python-pipelines-with-directrunner">
 <h2>Python Pipelines with DirectRunner<a class="headerlink" href="#python-pipelines-with-directrunner" title="Permalink to this headline">¶</a></h2>
 <div class="example-block-wrapper compound">
-<p class="example-header example-header--with-button compound-first"><span class="example-title">tests/system/providers/apache/beam/example_python.py</span><a class="example-header-button viewcode-button reference internal" href="_modules/tests/system/providers/apache/beam/example_python.html"><span class="viewcode-link">[source]</span></a></p>
+<p class="example-header example-header--with-button compound-first"><span class="example-title">tests/system/providers/apache/beam/example_python.py</span><a class="example-header-button viewcode-button reference internal" href="https://github.com/apache/airflow/tree/providers-apache-beam/4.0.0/tests/system/providers/apache/beam/example_python.py" target="_blank"><span class="viewcode-link">[source]</span></a></p>
```
[the remark this answers is not in the document]
Comment (potiuk) [reply docs-archive/apache-airflow-providers-apache-beam/4.0.0/operators.html:619]: Thanks. Good eyes. Fixing momentarily

[the remark this answers is not in the document]
Comment (potiuk) [reply docs-archive/apache-airflow-providers-apache-beam/4.0.0/operators.html:619]: Fix here https://github.com/apache/airflow/pull/24389/commits/7858a23232e6fb193e1dc7d2f98b0cda5ca2d4ac :facepalm:

**Comment by potiuk** [comment]
Fix applied @ashb

**Comment by potiuk** [comment]
Looks like the list of affected files is "about right" this time 

https://gist.github.com/potiuk/fca7dca641044a200a81b3ab236fba52

**Comment by potiuk** [comment]
BTW . As suspected, seems that the root cause was the autoapi upgrade https://github.com/apache/airflow/pull/20170 - the example links were missing since Airflow 2.2.4 where it was cherry-picked to and all providers affected were released after 9 Dec when it got merged.

**Comment by potiuk** [comment]
Hey @ashb - shall we merge it ? Seems all issues are fixed.

**Comment by potiuk** [comment]
(and users get 404s )

**Pull request merged**